cdip_connector/core/connector_base.py

```diff
@@ -75,20 +72,42 @@ async def extract_load(
 
             if extracted is not None:
                 self.logger.info(
-                    f"{integration.login}:{integration.id} {len(extracted)} recs to send"
+                    f"{integration.login}:{integration.id} extracted {len(extracted)} items",
+                    extra={
+                        "integration_id": integration.id,
+                        "extracted_count": len(extracted),
+                        "integration_type": integration.type_slug,
+                        "integration_name": integration.name,
+                        "integration_login": integration.login,
+
+                    }
                 )
-                if extracted:
-                    self.logger.info(f"first transformed payload: {extracted[0]}")
 
                 await self.load(session, extracted)
                 await self.update_state(session, integration)
 
                 total += len(extracted)
+
         if not total:
             self.logger.info(
-                f"{integration.login}:{integration.id} Nothing to send to SIntegrate"
+                f"{integration.login}:{integration.id} no new data.",
+                extra={
+                    "integration_id": integration.id,
+                    "extracted_count": total,
+                    "integration_type": integration.type_slug,
+                    "integration_name": integration.name,
+                    "integration_login": integration.login,
+                }
             )
-        return total
+
+        # Summary report for a single Integration.
+        return {
+            "integration_id": integration.id,
+            "extracted_count": total,
+            "integration_type": integration.type_slug,
+            "integration_name": integration.name,
+            "integration_login": integration.login,
+        }
```

Comment:
Rather than returning a list of integers (which was a decent eye-catcher in logs) I return a summary for what this execution produced. It's not perfect, but is more informative. And it might be useful in the short-term for a log based metric.